packages/apps/reputation-oracle/server/src/modules/abuse/abuse.service.ts

```diff
@@ -268,17 +296,20 @@ export class AbuseService {
             escrowAddress: escrowAddress,
             eventType: OutgoingWebhookEventType.ABUSE_DISMISSED,
           };
-          const webhookUrl = (
-            await OperatorUtils.getOperator(
-              chainId,
-              escrow.exchangeOracle as string,
-            )
-          ).webhookUrl as string;
+          const operator = await OperatorUtils.getOperator(
+            chainId,
+            escrow.exchangeOracle as string,
+          );
+          if (!operator || !operator.webhookUrl) {
+            throw new Error(
+              `Operator data is missing. Abuse id: ${abuseEntity.id}`,
+            );
+          }
```

Comment:
```suggestion
          if (!operator) {
            throw new Error('Operator data is missing');
          }
```
\+ extra check on webhookUrl if needed